modules/ROOT/pages/organization.adoc

```diff
@@ -124,16 +135,16 @@ When your organization has multiple business groups, you can navigate between th
 
 image::switch-suborg.png[switch+suborg]
 
-If your are not an organization administrator, you can only view the business groups that you have permissions to view. In the organization tab, the tree of your organization only displays the business groups you are a member of.
+If you are not an organization administrator, you can view only the business groups that you have permissions to view. In the organization tab, the tree of your organization displays only the business groups you are a member of.
```

Comment:
Great catch. Because people usually speak in the nongrammatical version, I always miss these!